torch/csrc/jit/python/python_tracer.cpp

```diff
@@ -23,25 +23,44 @@ namespace tracer {
 
 // Python interpreter retrieval routine adapted from
 // https://stackoverflow.com/a/8706144
-SourceRange getPythonInterpreterSourceRange() {
-  c10::optional<std::string> source_filename;
-  size_t source_line = 0;
-  std::stringstream stack_trace;
-
+std::vector<StackEntry> pythonCallstack() {
```

Comment:
nit: it's a bit weird that it lives in python_tracer (kind of unrelated), maybe move to some more generic file? (not critical)

Reply:
i'll check if there's a better location available, if not we'd prob. better not create a new one..